src/openms/source/FORMAT/MascotRemoteQuery.cpp

```diff
@@ -579,13 +579,27 @@ namespace OpenMS
 
       // see http://www.matrixscience.com/help/export_help.html for parameter documentation
       String required_params = "&do_export=1&export_format=XML&generate_file=1&group_family=1&peptide_master=1&protein_master=1&search_master=1&show_unassigned=1&show_mods=1&show_header=1&show_params=1&prot_score=1&pep_exp_z=1&pep_score=1&pep_seq=1&pep_homol=1&pep_ident=1&pep_expect=1&pep_var_mod=1&pep_scan_title=1&query_qualifiers=1&query_peaks=1&query_raw=1&query_title=1";
+      // TODO: check that export_params don't contain _show_decoy_report=1. This would add <decoy> at the top of the XML document and we can't easily distinguish the two files (target/decoy) from the search results later.
```

Comment:
worth mentioning that in the docs of `export_params` flag and maybe actually do the check?
Its only a single line of code.

Reply:
I could add it thou it doesn't help a lot as a user should not mess with the export_params anyway. Basically if you change them in any way (not only related to the decoys) you will likely break the export, so you must know in detail what you are doing.